python_modules/dagster-externals/dagster_externals/_context.py

```diff
@@ -23,23 +20,28 @@
     assert_param_type,
     assert_param_value,
     assert_single_asset,
+    param_from_env,
 )
+from .io.base import bootstrap_input_reader, bootstrap_output_writer
 
 
 def is_dagster_orchestration_active() -> bool:
-    return bool(os.getenv(DAGSTER_EXTERNALS_ENV_KEYS["is_orchestration_active"]))
+    return param_from_env("is_orchestration_active")
 
 
-def init_dagster_externals() -> "ExternalExecutionContext":
+def init_dagster_externals(
+    *,
+    input_reader: Optional[ExternalExecutionInputReader] = None,
+    output_writer: Optional[ExternalExecutionOutputWriter] = None
+) -> "ExternalExecutionContext":
```

Comment:
I think the use of "input" and "output" are a bit problematic here because they collide with dagster concepts of input and output. Let's discuss naming in our sync